tests/acceptance/channelQueryTest.php

```diff
@@ -69,31 +69,27 @@ public function testFilterByPrefixOneChannel()
 
     public function testUsersInfo()
     {
-        $response = $this->pusher->get_users_info('presence-channel-test');
+        $result = $this->pusher->get_users_info('presence-channel-test');
 
-        // print_r( $response );
+        // print_r( $result );
```

Comment:
Should probably just remove this!

Reply:
It's done all over the place, so I'll remove in a future PR